src/app/public/modules/code-examples/code-examples-editor.service.spec.ts

```diff
@@ -16,54 +16,180 @@ import {
+  });
+
+  it('should set SkyTheme to modern when theme property is set to Modern', () => {
+    codeExample.theme = SkyDocsCodeExampleTheme.Modern;
+
+    service.launchEditor(codeExample);
 
-  it('should provide SkyThemeService when theme property is set to Modern', () => {
-    const service = new SkyDocsCodeExamplesEditorService();
-    const stackblitzSpy = spyOn(StackBlitzSDK, 'openProject').and.callFake(() => {});
-    const codeExample: SkyDocsCodeExample = {
-      heading: 'foo',
-      packageDependencies: {},
-      sourceCode: [
-        {
-          fileName: 'foo.component.ts',
-          filePath: 'foo',
-          rawContents: 'These are my file contents!'
-        }
-      ],
-      theme: SkyDocsCodeExampleTheme.Modern
-    };
+    expect(stackblitzSpy).toHaveBeenCalled();
+    const spyArgs = stackblitzSpy.calls.mostRecent().args;
+    expect(spyArgs[0].files['src/app/app.component.ts']).toContain(`SkyTheme.presets['modern']`);
+    expect(spyArgs[0].files['src/app/app.module.ts']).toContain('SkyThemeService');
+  });
+
+  it('should set SkyTheme to default when theme property is set to Default', () => {
+    codeExample.theme = SkyDocsCodeExampleTheme.Default;
 
     service.launchEditor(codeExample);
 
     expect(stackblitzSpy).toHaveBeenCalled();
     const spyArgs = stackblitzSpy.calls.mostRecent().args;
-    expect(spyArgs[0].files['src/app/app.component.ts']).toContain('SkyThemeService');
+    expect(spyArgs[0].files['src/app/app.component.ts']).toContain(`SkyTheme.presets['default']`);
     expect(spyArgs[0].files['src/app/app.module.ts']).toContain('SkyThemeService');
   });
 
-  it('should NOT provide SkyThemeService when theme property is set to Default', () => {
-    const service = new SkyDocsCodeExamplesEditorService();
-    const stackblitzSpy = spyOn(StackBlitzSDK, 'openProject').and.callFake(() => {});
-    const codeExample: SkyDocsCodeExample = {
-      heading: 'foo',
-      packageDependencies: {},
-      sourceCode: [
-        {
-          fileName: 'foo.component.ts',
-          filePath: 'foo',
-          rawContents: 'These are my file contents!'
-        }
-      ],
-      theme: SkyDocsCodeExampleTheme.Default
-    };
+  it('should add modules from code example to app.module.ts', () => {
+    moduleImports.push('SampleDemoModule');
 
     service.launchEditor(codeExample);
 
     expect(stackblitzSpy).toHaveBeenCalled();
     const spyArgs = stackblitzSpy.calls.mostRecent().args;
-    expect(spyArgs[0].files['src/app/app.component.ts']).not.toContain('SkyThemeService');
-    expect(spyArgs[0].files['src/app/app.module.ts']).not.toContain('SkyThemeService');
+    expect(spyArgs[0].files['src/app/app.module.ts']).toContain(`import {\n  SampleDemoModule\n} from './foo.module';`);
+    expect(spyArgs[0].files['src/app/app.module.ts']).toContain(`imports: [\n    ${moduleImports.join(',\n    ')}`);
+  });
+
+  it('should throw an error if code example module does not have any exports', () => {
```

Comment:
👇 A lot of these tests aren't related to this PR. They were to get coverage to pass.